src/voices/index.ts

```diff
@@ -56,6 +57,27 @@ export default class Voices extends Client {
 		throw new Error("Invalid mode for clone()");
 	}
 
+	async createHifiClone(options: HifiCloneOptions): Promise<Voice> {
+		const formData = new FormData();
+		formData.append("clip", options.clip);
+		formData.append("name", options.name);
+		formData.append("description", options.description);
+		formData.append("language", options.language);
+		formData.append("model_id", options.model_id);
+		if (options.transcript) {
+			formData.append("transcript", options.transcript);
+		}
```

Comment:
Isn't this required for Hi-Fi cloning? Why is the user allowed to omit it?

Reply:
We'll generate it for them via whisper if they omit it.